### ISSUE TITLE
But where are the templates actually stored?

### ISSUE THREAD

**Issue by StingyJack** [dotnet/docs#23725]
So they can be forcefully removed if necessary, or otherwise "managed".

Specifically custom templates not the default ones. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2cd27054-288a-cd42-1f1d-9a58ef2b3a40
* Version Independent ID: 85ef1737-d703-ea92-7adb-71dc8c78a854
* Content: [Install and manage SDK templates - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/install/templates)
* Content Source: [docs/core/install/templates.md](https://github.com/dotnet/docs/blob/main/docs/core/install/templates.md)
* Product: **dotnet**
* Technology: **dotnet-install**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

**Comment by adegeo** [comment]
Hey there. Is there a specific question or clarification needed? I'm unsure what to do with this issue 😄

**Comment by StingyJack** [comment]
This page talks about managing templates but does so from the context of the provided tools, I was trying to find where the templates were actually installed to when installing a custom template so I could deal with scenarios where there was an installation problem. I found them a few minutes ago at 

%USERPROFILE%\.templateengine\dotnetcli\v5.0.201\packages


There are also templates in the %PROGRAMFILES%\dotnet\templ;ates folder. Since the page is about managing the templates I figured it would have path information as well. Up to you to include it, or leave this ticket as the place for that info.
